### ISSUE TITLE
Source line information are missing for some constructs in generated byte code

### ISSUE THREAD

**Issue by NipunaRanasinghe** [ballerina-platform/ballerina-lang#26975]
**Description:**
Please refer the example below. After debugging the below sample code, it could be observed that the source line information for the object methods of `Person` are missing as follows. (Here the source line information are available only for the lines within the main function). This affects source information depended components such as debugger and hence a blocker.

![Screenshot from 2020-11-17 13-24-13](https://user-images.githubusercontent.com/29032600/99361549-3f00dc00-28d8-11eb-8955-36eb4e27784b.png)

![Screenshot from 2020-11-17 13-24-02](https://user-images.githubusercontent.com/29032600/99361696-82f3e100-28d8-11eb-95a0-071e6b6e6611.png)

**Steps to reproduce:**
Use the sample code below.
```ballerina
public client class Person {
    public int age;
    public string firstName;
    public string lastName;

    function init(int age, string firstName, string lastName) {
        self.age = age;
        self.firstName = firstName;
        self.lastName = lastName;
    }

    function getFullName() returns string {
        return self.firstName + " " + self.lastName;
    }

    public remote function getName() returns string {
        string name = self.firstName;
        return name;
    }
}

public function main() {
    Person person = new Person(25, "John", "Doe");
    string fullName = person.getFullName();
    string name = person -> getName();
}
```

**Affected Versions:**
SLP6-SNAPSHOT
branch: `prj_api_module_refactor`

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by NipunaRanasinghe** [comment]
Please note that this works as expected in `master` branch and only exists in new project API branch (`prj_api_module_refactor`).

**Comment by NipunaRanasinghe** [comment]
Closing since fixed with #27126.
